5.2.Cup_Of_Join.py

```diff
@@ -0,0 +1,14 @@
+# def function Join that get unlimited number of lists and return a list of all the items in the lists
+# the function can recive an other parameter 'sep' that will be the separator between the items in the list
+# if the parameter is not given the default value will be '-'
+def join(*lists, sep='-'):
```

Comment:
Over-Documented. Use documentation to explain chunks of code or unclear lines. Not every line needs a comment.